CHANGELOG.md

```diff
@@ -73,6 +74,20 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Core
+
```

Comment:
we need to add apisix-base upgrade for fix CVE-2023-44487 https://github.com/apache/apisix/issues/10320

Reply:
@AlinsRan Just to confirm, no changes related to apisix-base needed in this repo right?

Reply:
Yes.